config/tests/chime_frb_verification.yaml

```diff
@@ -16,7 +16,7 @@ log_level: info
 num_elements: 2048
 num_local_freq: 1
 num_data_sets: 1
-samples_per_data_set: 49152
+samples_per_data_set: 4608
```

Comment:
Should we just leave this as it was?

Reply:
So originally this was set to `3840` to improve the speed. I changed it to `49152` as I added a check in `hsaBeamformHFBSum.cpp` to make sure it was a multiple of 12. But I should have just set it to `4608`.